cocos/platform/CCGLView.cpp

```diff
@@ -29,6 +29,8 @@ THE SOFTWARE.
 #include "base/CCDirector.h"
 #include "base/CCEventDispatcher.h"
 
+#include "2d/CCScene.h"
```

Comment:
GLView is a low level concepts and Scene is much high concepts.  The low level class shouldn't rely on the top.